### ISSUE TITLE
Second (useless) encryption performed in Proxy class

### ISSUE THREAD

**Issue by mhousser** [esbenp/lumen-api-oauth#3]
In Auth/Proxy.php, line 50:

```
$encryptedToken = $crypt->encrypt($response['refresh_token']);
```

This variable is never used, and a second encryption is performed a few lines later. Not causing any problems, but an inefficiency and technical mistake nonetheless.

**Comment by esbenp** [comment]
Ah yes, I will update the repo later on. Thanks!

**Comment by esbenp** [comment]
Fixed in c121bfda4046522e84e1af15f957cf7194e61292
